docs: :memo: post on how to write learning shorts (#19)
seedcase-project/community: guides/learning-shorts.qmd

---
title: "Writing a learning short for micro-learning"
description: "Learning shorts as a useful teaching tool when sharing knowledge and teaching others. This post briefly describes what a learning short is and how to write one."
date: "2024-05-15"
categories:
    - contributing
    - teaching shorts
    - microlearning
---

Part of the aim of the Seedcase Project isn't just to build software,
the aim is also to spread knowledge and teach others on concepts within
research data engineering, research software engineering, analytic
programming/coding, data management, and open research practices (like
FAIR data).

One tool we use as part of our teaching strategy is called
micro-learning or micro-lessons (which we call "learning shorts" or
"shorts"). Learning shorts are short, focused lessons that are designed
to teach a single concept or skill, with minimal or at least explicitly

> described outside knowledge.

Author: ```suggestion
described prerequisite knowledge.
```

["microlearning"](https://www.growthengineering.co.uk/what-is-microlearning/),
which is a method of teaching that breaks down learning content into
very small, very specific bits of information. The length of a typical
learning short is very short, only a few minutes long. We aim to keep
shorts under 10 minutes.

The concept of using microlearning for teaching arose acknowledging the
limited time we have in our modern, busy, information-overloaded world,
as well as from the science of learning that shows "less is more" when
it comes to long-term learning outcomes. Microlearning can also fit
nicely into a broader ["learning
campaign"](https://www.growthengineering.co.uk/learning-campaigns/)
strategy, where a series of connected and diverse learning activities
(like shorts, multi-day courses, or seminars) are used to achieve a
larger learning goal by encouraging behaviour and culture change over a
persistent, longer period of time.

## Learning shorts template

Within the `shorts/` folder of the learning section contains a template
for writing these shorts called `_template.qmd`. The template is a
Markdown file that contains the following structure:

> ``` markdown

Author: Good point!

campaign"](https://www.growthengineering.co.uk/learning-campaigns/)
strategy, where a series of connected and diverse learning activities
(like shorts, multi-day courses, or seminars) are used to achieve a
larger learning goal by encouraging behaviour and culture change over a
persistent, longer period of time.

## Learning shorts template

The `shorts/` folder of the learning section contains a template for
writing these shorts called `_template.qmd`. See the template to learn
about the structure.

You can refer to existing shorts to use as a guide and for inspiration
for how to write the individual sections. We won't go into too much
detail for each section, but will highlight a few:

-   **Learning goal**: This is the most important part of the short,
    aimed to inform the reader of what knowledge or skill will they
    gain. It should be as simple and short as needed to convey the
    learning goal. It should be written in a form based on [Bloom's
    Taxonomy](https://bloomstaxonomy.net/) (or a similar structure).
-   **Lesson content**: This is the main part of the short, where the
    core teaching content is written. It should be broken down into

>     sections (with several headers denoted by `###`). Try to make use of
>     images, diagrams, tables, and other visual aids to help explain the

Author: ```suggestion
    sections (with several headers denoted by `###`). Try to make use of
    visual aids like images, diagrams, and tables to help explain the
```

## Learning shorts template

The `shorts/` folder of the learning section contains a template for
writing these shorts called `_template.qmd`. See the template to learn
about the structure.

You can refer to existing shorts to use as a guide and for inspiration
for how to write the individual sections. We won't go into too much
detail for each section, but will highlight a few:

-   **Learning goal**: This is the most important part of the short,
    aimed to inform the reader of what knowledge or skill will they
    gain. It should be as simple and short as needed to convey the
    learning goal. It should be written in a form based on [Bloom's
    Taxonomy](https://bloomstaxonomy.net/) (or a similar structure).
-   **Lesson content**: This is the main part of the short, where the
    core teaching content is written. It should be broken down into
    sections (with several headers denoted by `###`). Try to make use of

>     images, diagrams, tables, and other visual aids to help explain the

Reviewer: Oh, I wonder what "other visual aids" could be? Videos?

>     content. If it is code based, make sure to actually show code.

Reviewer: ```suggestion
    content. If it is code based, make sure to show the actual code.
```
